connect/api/src/main/java/org/apache/kafka/connect/connector/Connector.java

```diff
@@ -82,6 +82,10 @@ public void initialize(ConnectorContext ctx, List<Map<String, String>> taskConfi
         // are very different, but reduces the difficulty of implementing a Connector
     }
 
+    public ConnectorContext context() {
```

Comment:
It'd be great to have JavaDoc for all API methods.

Reply:
Also, I wonder if it'd be better to have this be `protected` so that the connector subclass can access the context as now currently allowed by the protected field, but would prevent other objects from accessing it. We can always make it public in the future if we find we need that change, but once it's public we cannot retract it. WDYT?